README.md

```diff
@@ -45,6 +47,8 @@ In a basic flask app, This is our directory structure.
 
 All of our Python code lives in `app.py`, and to run the program we run `python app.py`. Static content lives in the `static` folder, and all HTML pages live in the `templates` folder.
 
+If you would like more review, check out ADI's [previous Flask tutorial](http://learn.adicu.com/webdev/).
```

Comment:
Awesome!  I might use `ADI's [introduction to Flask tutorial](http://learn.adicu.com/webdev/)` here, to be more clear